frontend/javascripts/oxalis/api/api_latest.js

```diff
@@ -1654,7 +1653,9 @@ class DataApi {
       }
     }
 
-    await loadMeshFromFile(segmentId, seedPosition, meshFileName, segmentationLayer, dataset);
+    Store.dispatch(
```

Comment:
It is used and awaited in the connectome viewer (via the cross origin api). However, as far as I remember this stems from a time, where deleting a mesh while it was loading led to errors. Therefore, it is no longer relevant to await the mesh loading and might even speed up the connectome viewer, because both meshes will be loaded in parallel instead of sequentially.
Since the mesh loading can take quite long, I'm also not sure whether the blocking nature of this function would add much value. I'll adapt the usages and doc-string.

Reply:
> It is used and awaited in the connectome viewer (via the cross origin api). However, as far as I remember this stems from a time, where deleting a mesh while it was loading led to errors. Therefore, it is no longer relevant to await the mesh loading and might even speed up the connectome viewer, because both meshes will be loaded in parallel instead of sequentially.

Two birds with one stone, I guess :tada: 

> Since the mesh loading can take quite long, I'm also not sure whether the blocking nature of this function would add much value. I'll adapt the usages and doc-string.

Sounds very plausible :+1: